core/store/src/test/java/uk/gov/gchq/gaffer/store/StoreTest.java

```diff
@@ -197,46 +203,36 @@ public void setup() {
         System.clearProperty(JSONSerialiser.JSON_SERIALISER_MODULES);
         JSONSerialiser.update();
 
-        schemaOptimiser = mock(SchemaOptimiser.class);
-        when(schemaOptimiser.optimise(any(Schema.class), any(Boolean.class))).then(returnsFirstArg());
-        operationChainValidator = mock(OperationChainValidator.class);
         store = new StoreImpl();
-        given(operationChainValidator.validate(any(OperationChain.class), any(User.class), any(Store.class))).willReturn(new ValidationResult());
-        addElementsHandler = mock(OperationHandler.class);
-        getElementsHandler = mock(OutputOperationHandler.class);
-        getAllElementsHandler = mock(OutputOperationHandler.class);
-        getAdjacentIdsHandler = mock(OutputOperationHandler.class);
-        validateHandler = mock(OperationHandler.class);
-        exportToGafferResultCacheHandler = mock(OperationHandler.class);
-        getGafferResultCacheExportHandler = mock(OperationHandler.class);
-        jobTracker = mock(JobTracker.class);
-        schema = new Schema.Builder()
-                .edge(TestGroups.EDGE, new SchemaEdgeDefinition.Builder()
-                        .source("string")
-                        .destination("string")
-                        .directed("true")
-                        .property(TestPropertyNames.PROP_1, "string")
-                        .property(TestPropertyNames.PROP_2, "string")
-                        .build())
-                .edge(TestGroups.EDGE_2, new SchemaEdgeDefinition.Builder()
-                        .source("string")
+        lenient().when(schemaOptimiser.optimise(any(Schema.class), any(Boolean.class))).then(returnsFirstArg());
```

Comment:
Again, why lenient?

Reply:
With `when` it errors if the mock was declared but not used as was the case with this. If you add lenient it does not error if a particular mock was not used.

Reply:
I guess this is fine for migrating to new mockito but probably means the tests should be refactored